aict_tools/apply.py

```diff
@@ -106,9 +107,21 @@ def create_mask_h5py(
         name, (operator, value) = list(c.items())[0]
 
         before = mask.sum()
-        mask = np.logical_and(
-            mask, OPERATORS[operator](infile[key][name][start:end], value)
-        )
+        if isinstance(infile, tables.file.File):
+            table = infile.get_node(key)
+            if name not in table.colnames:
```

Comment:
A column that is missing but required in the cuts is an error condition, not something that should be silently ignored.

At least the log level should be warning. But I just would raise an error.

Reply:
I think the reasoning for this is that not all selection columns are necessarily present in the same table. Still agree to a point.